mmeds/config.py

```diff
@@ -284,6 +288,12 @@
     except FileExistsError:
         pass
 
+if not SEQUENCING_DIR.exists():
+    try:
+        SEQUENCING_DIR.mkdir()
+    except FileExistsError:
+        pass
```

Comment:
why would the sequencing dir not exist and yet have a FileExistsError when trying to make it?

Reply:
Not sure, this is just how all the directories are created in `config.py`, wanted to keep it consistent. It must have been some error that David was running into when setting this up.

Reply:
Open issue: things that we don't understand need to be investigated.